CONTRIBUTING.md

```diff
@@ -86,3 +86,41 @@ described below:
   "proposal finalized" label.  This indicates that it is ready to be
   implemented by a platform developer, see the [process](#contributing) above.
 - Once a Pull Request is approved, it will be merged into the 'master' branch.
+
+## Spec Release Process
+
+Any member of the OSBAPI PMC can request that the accepted spec changes on
+master are released in a new version of the spec. They will do this by
+creating a new PR with the tile of the proposed release. For example,
+"Release Proposal: v2.20".
+
+### Prepare a PR
+
+- In a fork create a new branch called "v$major.$minor".
+- Update [README](README.md)'s "Latest Release" section and link to branch URL.
```

Comment:
As this will be mostly internal, I don't think it is a big deal if the link is broken at the beginning. Also, the spec that will be version bumped is the one in master, so there doesn't seem to be room for ambiguity if we don't directly link the right spec in the initial PR, is that right?